azext_iot/central/params.py

```diff
@@ -150,6 +150,13 @@ def load_central_arguments(self, _):
             " A case-sensitive string (up to 128 characters long) of ASCII 7-bit alphanumeric characters plus"
             " certain special characters: - . + % _ # * ? ! ( ) , : = @ $ '",
         )
+    with self.argument_context("iot central device manual-failover") as context:
+        context.argument(
+            "ttl_minutes",
+            type=int,
```

Comment:
Can we add some validation ranges to the arguments passed here? Right now the user can pass -1

Reply:
handled in controller layer